pkg/e2e/verify/namespace_webhook.go

```diff
@@ -254,6 +357,15 @@ func deleteNamespace(ctx context.Context, namespace string, waitForDelete bool,
 	return err
 }
 
+func deletePrometheusRule(namespace, prometheusName string, h *helper.H) error {
+	log.Printf("Deleting prometheusrule for prometheusrule validation webhook (%s)", prometheusName)
+	err := h.Prometheusop().MonitoringV1().PrometheusRules(namespace).Delete(context.TODO(), prometheusName, metav1.DeleteOptions{})
+	if err != nil {
```

Comment:
This will need to account for the situation that the PrometheusRule never existed in order to be deleted (which I'm _guessing_ should return an error). 

So something like (untested..):
```suggestion
	if err != nil && !errors.IsNotFound(err) {
```